CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Fixes
 
 - `[jest-core]` Fix typo in `scheduleAndRun` performance marker ([#14434](https://github.com/jestjs/jest/pull/14434))
+- `[jest-runner]` Handle unsafe worker_threads structureClone with function type in matchers. ([#14436](https://github.com/jestjs/jest/pull/14436))
+- `[jest-worker]` Additional error wrapper for `parentPort.postMessage` to fix unhandled `DataCloneError`. ([#14436](https://github.com/jestjs/jest/pull/14436))
```

Comment:
Could you split this into separate PR. It looks useful and could land sooner while the rest will be polished.

Reply:
Done https://github.com/jestjs/jest/pull/14437

I'll rebase and clean current PR after the merge